### ISSUE TITLE
"View details" modal needs content change

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4091]
## Area or Component
Results page - view details modal

## Quick Summary
There are two headers for "Test details" on the modal, but they describe different content - one should likely be renamed.

## Accessibility Violation
Heading does not describe content: https://cdc-devtools.dequecloud.com/rule-help/en/heading-not-descriptive

## Action Requested
Content ask: one of the two "test details" subheadings should be renamed if possible.

## Screenshots (optional)
<img width="802" alt="Screen Shot 2022-07-27 at 12 18 25 PM" src="https://user-images.githubusercontent.com/80282552/181354306-246315f3-100e-4c3b-b819-5848cf4f3624.png">

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1978534823&range=39:39

**Comment by lauraykerr** [comment]
@georgehager Without regrouping the info on this page (e.g. pulling out results into it's own section), would recommend: 

- Renaming the whole card "Result details"  — rationale: since it's the Test results page, each entry is a "result")
- Renaming the subsection "Test information" — not much rationale here since it's a mix of result info and test info, but it's different than the main header

It's imperfect but solves the immediate issue. @kenieh Looping you in in case you have other ideas!

**Comment by georgehager** [comment]
@lauraykerr I'll make this change for now, unless we think this warrants a bigger re-design. Maybe it would make sense for that to be a new ticket, though.

**Comment by georgehager** [comment]
@lauraykerr Does this look good to you?
![Screen Shot 2022-10-03 at 1 38 32 PM](https://user-images.githubusercontent.com/89797785/193642809-1e957b0c-b93e-4137-8235-1d871c45ebce.png)

**Comment by lauraykerr** [comment]
@georgehager LGTM, thank you!

And agreed about a new ticket for redesign! I can create that.

**Comment by kenieh** [comment]
LGTM as well, thanks for covering @lauraykerr and thanks @georgehager for looping us in.
